csharp/Facebook.Yoga.Desktop.sln

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
Added a solution for desktop, universal, and all.... The desktop version contains the desktop test project, but does not contain the universal test project.  The Facebook.Yoga solution contains both the desktop and universal tests...